schemas/2015-08-01/Microsoft.Automation.json

```diff
@@ -0,0 +1,145 @@
+{
+    "id": "http://schema.management.azure.com/schemas/2016-06-01/Microsoft.Automation.json#",
```

Comment:
Please update 2014-04-01-preview\deploymentParameters.json and 2015-01-01\deploymentTemplate.json to include the new resource type references.